### ISSUE TITLE
vsftpd fails to start with `unrecognised variable in config file: ssl_tlsv2`

### ISSUE THREAD

**Issue by hcooper** [hassio-addons/addon-ftp#108]
I've just been trying to run the github head version of this addon, and it just loops with

```
[17:50:28] INFO: Starting FTP server...
[17:50:29] INFO: Starting FTP server...
[17:50:30] INFO: Starting FTP server...
[17:50:31] INFO: Starting FTP server...
```

I think the issue might be [this change](https://github.com/hassio-addons/addon-ftp/commit/50f1b38a9f29952d53a4375f267afd27afeb4840), as when I try running vsftpd from inside the container it complains like this...

```
root@local-ftp:/$ /run/s6/etc/services.d/vsftpd/run
[17:48:47] INFO: Starting FTP server...
500 OOPS: unrecognised variable in config file: ssl_tlsv2
```

If I edit `/etc/vsftpd/vsftpd.conf` in the running container, and remove the new ssl related lines, vsftpd starts up fine.

**Comment by hcooper** [comment]
Oh, think the issue is a simple typo, e.g. `s/tlsv2/sslv2/'
``
ssl_sslv2
ssl_sslv3
ssl_tlsv1
```

Per https://security.appspot.com/vsftpd/vsftpd_conf.html

I'll put up PR with a fix.

**Comment by frenck** [comment]
fixed in #109
